core/iwasm/compilation/aot_emit_function.c

```diff
@@ -2534,6 +2565,12 @@ aot_compile_op_call_indirect(AOTCompContext *comp_ctx, AOTFuncContext *func_ctx,
         goto fail;
     }
 
+    if (comp_ctx->aot_frame
+        && comp_ctx->call_stack_features.frame_per_function) {
+        aot_alloc_frame_per_function_frame_for_aot_func(comp_ctx, func_ctx,
+                                                        func_idx);
```

Comment:
commit_params_to_frame_of_import_func was called in L2542, which assumes that the frame has been allocated, so here we should move these lines to be fore L2542?

And had better check the return value of aot_alloc_frame_per_function_frame_for_aot_func, if failed, goto fail